### PR TITLE
[stable/dex] add tolerations

### PR DESCRIPTION
**Pull request by vito-laurenza-zocdoc**
Signed-off-by: vito-laurenza-zocdoc <vito.laurenza@zocdoc.com>

@kfox1111 @sstarcher @rendhalver @desaintmartin 

#### What this PR does / why we need it:

Adds `tolerations` to the deployment template spec.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- ~~[ ] Variables are documented in the README.md~~ (`README.md` does not currently exist)
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/dex/templates/deployment.yaml [hunk 1]
```diff
@@ -37,6 +37,8 @@ spec:
       serviceAccountName: {{ template "dex.serviceAccountName" . }}
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 10 }}
+      tolerations:
```
Comment (desaintmartin) [inline stable/dex/templates/deployment.yaml:41]: I know this is not done this way for the other ones, but the code would be more resilient with :
```suggestion
{{- with .Values.tolerations }}
      tolerations:
{{ toYaml . | indent 8 }}
{{- end }}
```

**Comment by desaintmartin** [comment]
Thanks!
/ok-to-test

**Comment by desaintmartin** [comment]
lgtm when you sign-off the commits!
Thanks for the contribution.

**Comment by vito-laurenza-zocdoc** [comment]
> lgtm when you sign-off the commits!
> Thanks for the contribution.

They should both be signed off now. I followed the instructions from the `dco-labeler` bot to fix it. Not sure why it's still showing a check failure.

**Comment by desaintmartin** [comment]
Weird. Let's wait a few minutes.
In the meantime:
/lgtm

**Comment by desaintmartin** [comment]
Checks are passing but status is not updated by helm-bot, maybe you could try to rebase and force push to force update?

**Comment by vito-laurenza-zocdoc** [comment]
> Checks are passing but status is not updated by helm-bot, maybe you could try to rebase and force push to force update?

I squashed the commits and force pushed. That seemed to do the trick.

**Pull request merged**